holmscan/controller.py

```diff
@@ -22,24 +23,18 @@
 logging.getLogger("anyconfig").setLevel(logging.ERROR)
 
 
-CONFIGURABLES = [
-    "HOLMSCAN_DEBUG",
-    "HOLMSEC_PASSWORD",
-    "HOLMSEC_URL",
-    "HOLMSEC_USERNAME",
-]
+CONFIGURABLES = ["HOLMSCAN_DEBUG", "HOLMSEC_ENDPOINT", "HOLMSEC_TOKEN"]
 DEFAULTS = {
     "HOLMSCAN_DEBUG": False,
-    "HOLMSEC_PASSWORD": "",
-    "HOLMSEC_URL": "https://sc.holmsecurity.com/",
-    "HOLMSEC_USERNAME": "",
+    "HOLMSEC_ENDPOINT": "https://se-api.holmsecurity.com/v1",
+    "HOLMSEC_TOKEN": "",
 }
-REQUIRED = ["HOLMSEC_PASSWORD", "HOLMSEC_URL", "HOLMSEC_USERNAME"]
-VALIDATORS = {"HOLMSEC_URL": "^http(s)?://[a-zA-Z0-9._-]+/$"}
-VALID_EXAMPLES = {"HOLMSEC_URL": "example: https://sc.holmsecurity.com/"}
+REQUIRED = ["HOLMSEC_ENDPOINT", "HOLMSEC_TOKEN"]
+VALIDATORS = {"HOLMSEC_ENDPOINT": "^http(s)?://[a-zA-Z0-9._-]+/v[0-9]$"}
+VALID_EXAMPLES = {"HOLMSEC_ENDPOINT": "example: https://se-api.holmsecurity.com/v1"}
 CONFIG_EXAMPLES = {
-    "HOLMSEC_USERNAME": "example: export HOLMSEC_USERNAME=username@example.com",
-    "HOLMSEC_URL": "example: echo HOLMSEC_URL: https://sc.holmsecurity.com/ >> ~/.config/holmscan.yaml",
+    "HOLMSEC_ENDPOINT": "example: echo HOLMSEC_ENDPOINT: https://se-api.holmsecurity.com/v1 >> ~/.config/holmscan.yaml",
```

Comment:
`"HOLMSEC_ENDPOINT"` looks like a constant. Move it to
```
HOLMSEC_ENDPOINT = "HOLMSEC_ENDPOINT"
```

Reply:
I don't get the added benefit of this either.